src/test/java/util/ParserTest.java

```diff
@@ -0,0 +1,91 @@
+package util;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import exceptions.InvalidDateException;
+import exceptions.MizzException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+public class ParserTest {
+    @TempDir
+    Path tempDir;
+
+    @Test
+    public void testParseFromStorage_withValidData() {
```

Comment:
I believe all the methods here should be in camelCase. Let me know if I'm mistaken on this, since I don't believe the coding standard explicitly mentions the standard for test cases.
https://se-education.org/guides/conventions/java/intermediate.html#naming